### PR TITLE
Bug 1123188 - [Flame][Dialer]It can't enter corresponding settings page

### PR DESCRIPTION
**Pull request by EragonJ**
when you tap "Voicemail" and "Fixed Dialing Numbers".

### CODE REVIEW

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f72c5a41b70e)

File: apps/settings/js/panels/call_iccs/call_icc_handler.js [hunk 1]
```diff
@@ -0,0 +1,277 @@
+define(function(require) {
+  'use strict';
+
+  var DsdsSettings = require('dsds_settings');
+
+  /**
+   * Singleton object that helps to populate and manage the 'Select a SIM card'
+   * panel in the call settings panel.
+   */
+  var CallIccHandler = (function(window, document, undefined) {
```
Comment (crh0716) [inline apps/settings/js/panels/call_iccs/call_icc_handler.js]: nit: Please remove the third parameter.

File: apps/settings/js/activity_handler.js [hunk 2]
```diff
@@ -66,6 +66,12 @@
           // Apply the filter
           document.body.dataset.filterBy =
             activitySource.data.filterBy || 'all';
+        } else if (targetPanelId === 'call') {
+          var mozMobileConnections = navigator.mozMobileConnections;
```
Comment (crh0716) [inline apps/settings/js/activity_handler.js]: Please add a test case for this.

File: apps/settings/js/panels/call_iccs/call_icc_handler.js (hunk 1 above)
Comment (EragonJ) [inline apps/settings/js/panels/call_iccs/call_icc_handler.js]: sure okay !

File: apps/settings/js/activity_handler.js (hunk 2 above)
Comment (EragonJ) [inline apps/settings/js/activity_handler.js]: okay !

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6ce6c84b748a)

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cc0fdfd38742)

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=33e154023001)

**Pull request merged**